spec/form/work_form_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+
+describe Sufia::Forms::WorkForm do
```

Comment:
ok so in my understanding, I am overriding the multiple? in sufia's  work_form and I should be testing sufia work_form?